city_scrapers/spiders/stl_port_authority.py

```diff
@@ -0,0 +1,203 @@
+        else:
+            meeting["links"] = []
+
+        meeting["status"] = self._get_status(meeting)
+        meeting["id"] = self._get_id(meeting)
+        return meeting
+
+    def _parse_title(self, response):
+        """Parse or generate meeting title."""
+        title = response.css("div.page-title-row h1::text").get().strip()
+        return title
+
+    def _parse_start(self, response):
+        """Parse start datetime as a naive datetime object."""
+        date = response.css("div.page-title-row p.page-summary::text").get()
+        pattern = r"(?P<day>\d{2}/\d{2}/\d{2}), (?P<time>(\d{1,2}:\d{2}) (PM|AM))"
+        pattern += r" - (\d{1,2}:\d{2}) (PM|AM)"
+        rm = re.search(pattern, date)
+
+        if rm is not None:
+            day = rm.group("day")
+            time = rm.group("time")
+            dt = day + " " + time
+            start = datetime.strptime(dt.strip(), "%m/%d/%y %H:%M %p")
+            return start
+        else:
+            return None
+
+    def _parse_end(self, response):
+        """Parse end datetime as a naive datetime object. Added by pipeline if None"""
+        date = response.css("div.page-title-row p.page-summary::text").get()
+        pattern = r"(?P<day>\d{2}/\d{2}/\d{2}), (\d{1,2}:\d{2}) (PM|AM)"
+        pattern += r" - (?P<time>(\d{1,2}:\d{2}) (PM|AM))"
+        rm = re.search(pattern, date)
+
+        if rm is not None:
+            day = rm.group("day")
+            time = rm.group("time")
+            dt = day + " " + time
+            end = datetime.strptime(dt.strip(), "%m/%d/%y %H:%M %p")
+            return end
+        else:
+            return None
+
+    def _parse_all_day(self, response):
+        """Parse or generate all-day status. Defaults to False."""
+        return False
+
+    def _parse_location(self, response):
+        """Parse or generate location."""
+        location = response.css("div.col-md-4 div.content-block p *::text").getall()
+        title = response.css("div.page-title-row h1::text").get().strip()
+
+        temp = []
+        for item in location:
+            item = item.replace("\n", "")
+            if item != "":
+                temp.append(item)
+        location = temp
+        i, location_index, sponsor_index = 0, 0, 0
+        while i < len(location):
+            if "location" in location[i].lower():
+                location_index = i
+            if "sponsor" in location[i].lower():
+                sponsor_index = i
+                break
+            i += 1
+
+        if "zoom" in title.lower():
+            name = "Zoom"
+            address = ""
+        elif "phone" in title.lower():
+            name = "Phone"
+            address = ""
+        elif location_index + 1 < len(location) and sponsor_index < len(location):
+            name = location[location_index + 1]
+            address = []
+            for j in range(location_index + 2, sponsor_index):
+                address.append(location[j])
```

Comment:
It's a small detail, but could you fix the spacing in the address? I usually do [this](https://github.com/stl-public-meetings/city-scrapers-stl/blob/38904a3fd57549cbfba560344ceca530c4659211/city_scrapers/spiders/stl_development.py#L168-L171).